teachertool/src/components/EditorContainer.tsx

```diff
@@ -22,10 +22,7 @@ export const EditorContainer: React.FC<{}> = () => {
     }
```

Comment:
I think you can remove `onIframeLoaded` and `onIframeClosed` now, too

Reply:
(And possibly move createIFrameUrl into a util function)

Reply:
I think you're right. With moving these things, I'm thinking that we actually won't need the editor container at all, then, and can just plug the `MakeCodeFrame` straight into the app.

Reply:
I'm going to do this. I think any logic we'll possibly want to plug into the frame, it should be handled in the transform or the service.